dataactbroker/handlers/interfaceHolder.py

```diff
@@ -25,35 +25,5 @@ def __init__(self):
 
     def close(self):
         """ Close all open connections """
-        InterfaceHolder.closeOne(self.jobDb)
-        InterfaceHolder.closeOne(self.errorDb)
-        InterfaceHolder.closeOne(self.userDb)
-        InterfaceHolder.closeOne(self.validationDb)
-
-    @staticmethod
-    def closeOne(interface):
-        """ Close all aspects of one interface """
-        if(interface == None):
-            # No need to close a nonexistent connection
-            return
-        try:
-            if(interface.session == None):
-                # If session is None, skip closing
-                return
-        except AttributeError as e:
-            # If interface has no session, skip closing
-            return
-
-        # Try to close the session and connection, on error try a rollback
-        try:
-            interface.session.close()
-        except:
-            try:
-                interface.session.rollback()
-                interface.session.close()
-            except Exception as e:
-                exc_type, exc_obj, exc_tb = sys.exc_info()
-                trace = traceback.extract_tb(exc_tb, 10)
-                CloudLogger.logError('Broker DB Interface Error: ', e, trace)
-                del exc_tb
-                raise
+        if self.jobDb is not None:
+            self.jobDb.close()
```

Comment:
I will change the cleanup code in app.py to call this, the idea is to have that this repeated in different places but to instead always redirect it to the BaseInterface.close() method